### ISSUE TITLE
Version 1.3.0 and above are broken

### ISSUE THREAD

**Issue by sindresorhus** [zeit/hazel#26]
I deployed the latest version using:

```
now -e NODE_ENV="production" -e ACCOUNT="wulkano" -e REPOSITORY="kap" -e PORT="80" -e PRE="1" zeit/hazel
```

And to my surprise the result of going to https://kap-beta-updates.now.sh/update/macos/2.0.0-beta.0 was:

```json
{"name":"v2.0.0-beta.1","notes":"Test","pub_date":"2017-10-31T14:41:53Z","url":"https://api.github.com/repos/wulkano/kap/releases/assets/5210625"}
```

Notice how the `url` links to the GitHub asset response and not the binary itself.

With 1.2.2 it works correctly:

```json
{"name":"v2.0.0-beta.1","notes":"Test","pub_date":"2017-10-31T14:41:53Z","url":"https://github.com/wulkano/kap/releases/download/v2.0.0-beta.1/Kap.Beta-2.0.0-beta.1-mac.zip"}
```

So 1.2.2 works correctly, but 1.3.0 does not, so something must have broken between those versions.

**Comment by leo** [comment]
Should be fixed in [1.3.3](https://github.com/zeit/hazel/releases/tag/1.3.3) - please confirm!

**Comment by sindresorhus** [comment]
Yes, works. Thanks for fixing :)
